src/test/java/io/specto/hoverfly/junit/core/model/JournalTest.java

```diff
@@ -0,0 +1,37 @@
+package io.specto.hoverfly.junit.core.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import org.junit.Test;
+
+import java.net.URL;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JournalTest {
+
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private URL resource = Resources.getResource("sample-journal-logs.json");
+
+    @Test
+    public void shouldDeserializeFromJson() throws Exception {
+
+        Journal journal = objectMapper.readValue(resource, Journal.class);
+
+
+        assertThat(journal.getEntries()).hasSize(1);
+
+        JournalEntry logEntry = journal.getEntries().iterator().next();
+
+        assertThat(logEntry.getMode()).isEqualTo("simulate");
+        assertThat(logEntry.getRequest()).isNotNull();
```

Comment:
We need to actually assert the whole journal response in a test somewhere, otherwise we wouldn't know if it was broken.